BlogAPI.Persistence/Migrations/20240114210727_Initial.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace BlogAPI.Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class Initial : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "BlogPosts",
+                columns: table => new
+                {
+                    Id = table.Column<int>(type: "int", nullable: false)
+                        .Annotation("SqlServer:Identity", "1, 1"),
+                    Title = table.Column<string>(type: "nvarchar(50)", maxLength: 50, nullable: false),
+                    Author = table.Column<string>(type: "nvarchar(50)", maxLength: 50, nullable: false),
+                    Content = table.Column<string>(type: "nvarchar(max)", maxLength: 5000, nullable: false),
+                    CreationDate = table.Column<DateTime>(type: "datetime2", nullable: false),
+                    Tags = table.Column<string>(type: "nvarchar(max)", nullable: false)
+                },
+                constraints: table =>
+                {
+                    table.PrimaryKey("PK_BlogPosts", x => x.Id);
+                });
+
+            migrationBuilder.CreateTable(
+                name: "Comments",
```

Comment:
This could be discussed, but for the table names, what sort of convention are you going for. By the looks of it you are going with plural names as well as PascalCase. I'd like it more of it was singular names and snake_case for the name itself. What do you think @PauliusLozys

Reply:
Personally, I think having singular snake_case is better. Snake case is easier to read in SQL queries as they don't stand out from important SQL keywords like SELECT/WHERE/GROUP BY etc. Singular tables names also seem better as table contains many objects of singular nature. But in the need, I must say it's just preference, the most important part here is to be consistent with naming.

Reply:
I'll go with plural default generated by EF Core